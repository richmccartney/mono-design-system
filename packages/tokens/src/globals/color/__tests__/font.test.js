const fontColorTokens = require('../font.json');

describe('Font color tokens', () => {
  test(': matches snapshot', () => {
    expect(fontColorTokens).toMatchSnapshot();
  });
});
