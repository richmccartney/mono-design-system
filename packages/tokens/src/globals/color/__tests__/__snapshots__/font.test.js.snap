// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Font color tokens : matches snapshot 1`] = `
Object {
  "color": Object {
    "font": Object {
      "base": Object {
        "value": "{color.base.black.value}",
      },
      "primary": Object {
        "value": "{color.primary.value}",
      },
      "secondary": Object {
        "value": "{color.secondary.value}",
      },
      "tertiary": Object {
        "value": "{color.base.gray.value}",
      },
    },
  },
}
`;
